tests/zenko_e2e/backbeat/tests/crr-pause-resume/awsBackend.js

```diff
@@ -0,0 +1,116 @@
+const awsUtils = new ReplicationUtility(awsS3Client);
+const backbeatAPIUtils = new BackbeatAPIUtility();
+const srcBucket = `source-bucket-${Date.now()}`;
+const destBucket = process.env.AWS_S3_BACKBEAT_BUCKET_NAME;
+const destLocation = process.env.AWS_S3_BACKEND_DESTINATION_LOCATION;
+const hex = crypto.createHash('md5')
+    .update(Math.random().toString())
+    .digest('hex');
+const keyPrefix = `${srcBucket}/${hex}`;
+const key = `${keyPrefix}/object-to-replicate-${Date.now()}`;
+const key2 = `${key}-2`;
+// eslint-disable-next-line
+const REPLICATION_TIMEOUT = 300000;
+
+describe('Replication Pause-Resume with AWS backend', function() {
+    this.timeout(REPLICATION_TIMEOUT);
+    let roleArn = 'arn:aws:iam::root:role/s3-replication-role';
+
+    beforeEach(done => series([
+        next => scalityUtils.createVersionedBucket(srcBucket, next),
+        next => scalityUtils.putBucketReplicationMultipleBackend(srcBucket,
+            destBucket, roleArn, destLocation, next),
+    ], done));
+
+    afterEach(done => series([
+        next => scalityUtils.deleteVersionedBucket(srcBucket, next),
+        next => awsUtils.deleteAllVersions(destBucket,
+            `${srcBucket}/${keyPrefix}`, next),
+    ], done));
+
+    after(done => backbeatAPIUtils.resumeReplication(null, null, null, done));
+
+    it('should pause and resume replication', done => series([
+        next => scalityUtils.putObject(srcBucket, key, Buffer.alloc(1), next),
+        next => scalityUtils.waitUntilReplicated(srcBucket, key,
+            undefined, next),
+        next => backbeatAPIUtils.pauseReplication(null, next),
+        next => backbeatAPIUtils.getReplicationStatus(null, (err, data) => {
+            assert.ifError(err);
+            assert(data[destLocation]);
+            assert.strictEqual(data[destLocation], 'disabled');
+            return next();
+        }),
+        next => scalityUtils.putObject(srcBucket, key2, Buffer.alloc(1),
+            next),
+        next => awsUtils.assertNoObject(destBucket, key2, next),
+        next => backbeatAPIUtils.resumeReplication(null, null, null, next),
+        next => backbeatAPIUtils.getReplicationStatus(null, (err, data) => {
+            assert.ifError(err);
+            assert(data[destLocation]);
+            assert.strictEqual(data[destLocation], 'enabled');
+            return next();
+        }),
+        next => scalityUtils.waitUntilReplicated(srcBucket, key2,
+            undefined, next),
+    ], done));
+
+    it('should retrieve pause status', done => series([
+        next => backbeatAPIUtils.pauseReplication(null, next),
+        next => backbeatAPIUtils.getReplicationStatus(null, (err, data) => {
+            assert.ifError(err);
+            assert(data[destLocation]);
+            assert.strictEqual(data[destLocation], 'disabled');
+            return next();
+        }),
+    ], done));
+
+    it('should retrieve resume status', done => series([
```

Comment:
Same, it looks like it's duplicating the first test which is more complete. I would opt to remove it.

Reply:
I was hoping that life would be easier if I had to remove tests instead of make up tests :sweat_smile: , so I wrote them even though I was not 100% about necessity. @philipyoo @bennettbuchanan opinions on removal of the pause + resume status checks?